### PR TITLE
dont set the default schema when remove password form titles

### PR DESCRIPTION
**Pull request by zhijian-pro**
fix #3256

### CODE REVIEW

File: pkg/utils/utils.go [hunk 1]
```diff
@@ -101,6 +103,32 @@ func RemovePassword(uri string) string {
...
+				os.Args[i] = uri2
+				break
+			}
+		}
+	}
+	gspt.SetProcTitle(strings.Join(os.Args, " "))
+}
+
+func RemoveObjPwdFromTitle(uri string) {
+	removePwdFromTitle(uri, "file")
```
Comment (SandyXSD) [inline pkg/utils/utils.go:125]: may be 'file' or 'sftp'?

File: pkg/utils/utils.go [hunk 2]
```diff
@@ -101,6 +103,32 @@ func RemovePassword(uri string) string {
 	return uri[:sp+cp] + ":****" + uri[p:]
 }
 
+func removePwdFromTitle(uri, defaultSchema string) {
```
Comment (SandyXSD) [inline pkg/utils/utils.go:106]: IMO it's better to delete 'FromTitle'

File: pkg/utils/utils.go (hunk 1 above)
Comment (zhijian-pro) [reply pkg/utils/utils.go:125]: default  object storage is `file`

File: cmd/main.go [hunk 3]
```diff
@@ -293,21 +293,3 @@ func setup(c *cli.Context, n int) {
 		}
 	}
 }
-
-func removePassword(uri string) {
-	var uri2 string
-	if strings.Contains(uri, "://") {
-		uri2 = utils.RemovePassword(uri)
-	} else {
-		uri2 = utils.RemovePassword("redis://" + uri)
```
Comment (davies) [inline cmd/main.go:302]: Why we add this? Maybe just remove it.

**Pull request merged**